tests/utils.go

```diff
@@ -713,6 +713,7 @@ func CalculateNamespaces() {
 	testNamespaces = []string{NamespaceTestDefault, NamespaceTestAlternative, NamespaceTestOperator}
 }
 
+// CHECK THIS
```

Comment:
Is this an unhandled TODO marker?

Reply:
thanks for catching that! it's been removed.